boilerplate/package.json

```diff
@@ -117,10 +116,11 @@
     "react-devtools-core": "4.24.7",
     "react-dom": "18.1.0",
     "react-native-web": "~0.18.7",
+    "react-test-renderer": "18.2.0",
     "reactotron-core-client": "^2.8.10",
     "regenerator-runtime": "^0.13.4",
-    "ts-jest": "26",
-    "typescript": "^4.6.3",
+    "ts-jest": "29",
+    "typescript": "^4.8.4",
     "webpack": "4",
     "webpack-dev-server": "^3"
```

Comment:
Hi @frankcalise 👋 

`@expo/webpack-config` is using Webpack 5 now since it was updated in https://github.com/expo/expo-cli/pull/3763 

I was able to run my web build fine after updating to webpack 5 with similar warnings from the ones you got

```suggestion
    "webpack": "5",
    "webpack-dev-server": "^4"
```

Reply:
Thanks @carlosa54! Yeah I forgot they updated to webpack 5, will take a look at getting this updated.

Reply:
I still get those issues along with: 

```
WARNING in ./node_modules/expo-asset/build/ReactNativeCompatibleAssetsRegistry.js:7:13
Module not found: Can't resolve '@react-native/assets-registry/registry'
   5 | let registry: any | null = null;
   6 | try {
>  7 |   registry = require('@react-native/assets-registry/registry');
     |             ^
   8 | } catch {}
   9 | if (!registry) {
  10 |   try {
```

Reply:
Yes, I'm getting the same warning but it's not affecting functionality on my end. This is happening because of the optional require enclosed in the `try/catch`, webpack is still trying to resolve it and throws a warning. I wonder why it didn't failed in the previous webpack config.

Here's an old issue https://github.com/webpack/webpack/issues/7713#issuecomment-467888437

Reply:
Ok found out why and added a comment [here](https://github.com/expo/expo/issues/21276#issuecomment-1445417735)

Reply:
You get this assets-registry warning but expo web still runs ok? I'm getting trapped in the ErrorBoundary for max depth exceeded. Although maybe it isn't related?

Reply:
Exactly it's working fine on my project. I can pull this branch later today to test it

Reply:
Got it working on my end on this branch. For some weird reason the app is rendering twice for me

<img width="1288" alt="image" src="https://user-images.githubusercontent.com/4250593/221668465-5747a066-dbed-4244-866f-c105f093dc73.png">

Try adding this missing plugin to `babel.config.js`
[babel-config.patch](https://github.com/infinitered/ignite/files/10843466/babel-config.patch)
<img width="618" alt="image" src="https://user-images.githubusercontent.com/4250593/221668853-e014ab26-19ae-42f0-a51c-39120377c47f.png">

Reply:
Ok I got back to this as well. I did fix this double render issue once before I have to recall what I did.

Reply:
It's because of the `main` entry point in `package.json` being `App.js` - I think Expo 48 wants to work off of `index.js` like a regular app now. So it's probably launching both with the Ignite boilerplate having Expo or vanilla available.

If you copy the contents of `App.js` and paste it in `index.js`, remove `App.js` and remove the `main` line in `package.json, it should clear things up for you. Now I just have to figure out how to rework this.